std/parallelism.d

```diff
@@ -2864,13 +2864,13 @@ public:
             }
         }
 
-        ref T opIndex(size_t index)
+        ref T opIndex(size_t index) const
```

Comment:
More specifically, the cast in the function body could be abused by invoking the function on an immutable object (which is valid because immutable implicitly converts to const), thereby breaking immutability.